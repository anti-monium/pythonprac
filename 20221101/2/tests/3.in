A = Triangle((2,-1), (3,1), (-2,0))
B = Triangle((0,0), (0,0), (0,0))
C = Triangle((-3,1), (-2,2), (-1,3))
D = Triangle((2,0), (5,1), (1,3))
E = Triangle((2,1), (3,1), (2,2))
print(B in A, B & A)
print(E in D, E & D)
if bool(C):
	print('Hello, C!')
else:
	print('Where is C?')
print(A > B, B > C, C > D, D > E)
print(abs(A), abs(B))
if not abs(C):
	print('meow')
