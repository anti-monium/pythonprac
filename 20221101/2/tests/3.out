True False
True True
Where is C?
True False False True
4.5 0
meow
